src/test/java/com/neo4j/docker/coredb/TestMounting.java

```diff
@@ -345,4 +348,23 @@ void shouldReownSubfilesToNeo4j() throws Exception {
             // if debug.log doesn't get re-owned, neo4j will not start and this test will fail here
         }
     }
+
+    @Test
+	void shouldLoadPluginsFromMountedFolder() throws Exception {
+		Path testOutputFolder = temporaryFolderManager.createTempFolder( "mount-plugins-only-" );
+		try(GenericContainer container = setupBasicContainer( true, false )){
+			temporaryFolderManager.createTempFolderAndMountAsVolume( container, "plugins", "/plugins", testOutputFolder );
+			container.withEnv( "NEO4J_PLUGINS", "[\"apoc\"]" );
+			container.start();
+
+			assertApocIsLoaded(container);
```

Comment:
We should also be testing that the plugins folder created in line 356 actually contains apoc.jar.